x-pack/filebeat/module/checkpoint/firewall/ingest/pipeline.yml

```diff
@@ -481,6 +509,18 @@ processors:
       field: checkpoint.origin
       target_field: observer.name
       ignore_missing: true
+  - rename:
+      field: checkpoint.mac_address
+      target_field: observer.mac
+      ignore_missing: true
+  - gsub:
+      field: observer.mac
+      ignore_missing: true
+      pattern: '[:]'
+      replacement: '-'
+  - uppercase:
+      field: observer.mac
+      ignore_missing: true
```

Comment:
This can be removed after #32265 is merged, though maybe keeping it for parity with the integration is worthwhile.